gammapy/datasets/fermi.py

```diff
@@ -11,6 +12,7 @@
 from ..data import SpectralCube
 from ..datasets import get_path
 
+
 __all__ = ['FermiGalacticCenter',
            'FermiVelaRegion',
            'fetch_fermi_catalog',
```

Comment:
I'd say `Fermi3FGLObject` should be added to `__all__` here, so that it shows up in the API docs.
Otherwise it's hard for the user to discover it and figure out how to use it.